### ISSUE TITLE
Can't send Pulse - Unrecognized token 'DateTimeWithLocalTZ'

### ISSUE THREAD

**Issue by RoyHP** [metabase/metabase#12403]
**Describe the bug**
We created a new pulse, added some saved questions, and hit send email now. It says send failed and this showed up in our log. Running 0.35.3

**Logs**
POST /api/pulse/test 500 1.7 s (48 DB calls)
{:message "Unrecognized token 'DateTimeWithLocalTZ': was expecting ('true', 'false' or 'null')\n at [Source: (StringReader); line: 1, column: 20]",
:type com.fasterxml.jackson.core.JsonParseException,
:stacktrace

**To Reproduce**
Create pulse wizard
Add 3 questions
Add 2 email addresses
Click send now

**Expected behavior**
Should have sent the pulse email

**Screenshots**
<img width="1274" alt="Screen Shot 2020-04-27 at 12 56 58 PM" src="https://user-images.githubusercontent.com/1066439/80399188-cc220c00-8886-11ea-8dd3-535e12b568b0.png">

**Information about your Metabase Installation:**
docker image, v0.35.3

`{
  "browser-info": {
    "language": "en-US",
    "platform": "MacIntel",
    "userAgent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.122 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "11.0.5+10",
    "java.vendor": "AdoptOpenJDK",
    "java.vendor.url": "https://adoptopenjdk.net/",
    "java.version": "11.0.5",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "11.0.5+10",
    "os.name": "Linux",
    "os.version": "4.14.106-97.85.amzn2.x86_64",
    "user.language": "en",
    "user.timezone": "GMT"
  },
  "metabase-info": {
    "databases": [
      "mysql"
    ],
    "hosting-env": "unknown",
    "application-database": "mysql",
    "application-database-details": {
      "database": {
        "name": "MySQL",
        "version": "5.7.12"
      },
      "jdbc-driver": {
        "name": "MariaDB Connector/J",
        "version": "2.5.1"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2020-04-21",
      "tag": "v0.35.3",
      "branch": "release-0.35.x",
      "hash": "1d424cb"
    },
    "settings": {
      "report-timezone": "US/Eastern"
    }
  }
}`

**Severity**
We can't schedule any pulses, which is a major thing we want to do.

**Additional context**
The questions didn't seem to load when they were added - they had a loader on them that kept spinning.

**Comment by flamber** [comment]
Hi @RoyHP 
It looks very similar to #12330 and perhaps related to #11665.
Are the questions Native/SQL? If you create a Simple question based on the existing SQL in Saved Question, does that make it work? https://github.com/metabase/metabase/issues/12330#issuecomment-616994537

**Comment by RoyHP** [comment]
@flamber We are using the custom builder (middle option) - no native SQL.

**Comment by flamber** [comment]
@RoyHP 
Are you querying MySQL (same version as the application database, 5.7.12)?
Can you post the MySQL table structure/schema?

**Comment by alioSHA512** [comment]
I've ran into similar problem and find a way to fix/workaround. 

Recently we've updated from `0.33.x` to `0.38.x`. After that, a pulse failed to send everytime, with error like:

```
"Unrecognized token 'avg': was expecting (JSON String, Number, Array, Object or token 'null', 'true' or 'false')\n at [Source: (StringReader); line: 1, column: 4]"
```

With the ID of the question for `report_card.id`, I've managed to find that `avg` in `report_card.visualization_settings` (more precisely, in `visualization_settings["column_settings"]`) which leads to the parse error. 

In our case, the `avg` may exist becuz previously we use auto-generated column name (which contains the function name `avg`) and made column settings. Even though we've changed the column_name, the old settings' still there.

Modify `visualization_settings` and remove the k-v with `avg` works for us.

I recon this won't fix the JSON parse problem, but _change column name in the SQL then remove the problematic one in settings_ may be a workaround?
